docs-2.0/nebula-operator/3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md

```diff
@@ -51,59 +51,184 @@ The following example shows how to create a NebulaGraph cluster by creating a cl
+        |`spec.imagePullSecrets`| - |Specifies the Secret for pulling the NebulaGraph Enterprise service images from a private repository.|
+
+
+    === "Cluster with Zones"
+
+        NebulaGraph Operator supports creating a cluster with [Zones](../../4.deployment-and-installation/5.zone.md). 
+        
+        You must set the following parameters for creating a cluster with Zones. Other parameters can be changed as needed. For more information on other parameters, see the [sample configuration](https://github.com/vesoft-inc/nebula-operator/blob/v{{operator.release}}/config/samples/apps_v1alpha1_nebulacluster.yaml). 
+
+        | Parameter    | Default value  | Description    |
+        | :---- | :--- | :--- |
+        | `spec.metad.licenseManagerURL` | - | Configure the URL that points to the LM, which consists of the access address and port number (default port `9119`) of the LM. For example, `192.168.8.100:9119`. **You must configure this parameter in order to obtain the license information; otherwise, the enterprise edition cluster cannot be used.** |
+        |`spec.<graphd|metad|storaged>.image`|-|The container image of the Graph, Meta, or Storage service of the enterprise edition.|
+        |`spec.imagePullSecrets`| - |Specifies the Secret for pulling the NebulaGraph Enterprise service images from a private repository.|
+        |`spec.alpineImage`|`reg.vesoft-inc.com/nebula-alpine:latest`|The Alpine Linux image, used to obtain the Zone information where nodes are located.| 
+        |`spec.metad.config.zone_list`|-|A list of zone names, split by comma.|
+        |`spec.graphd.config.prioritize_intra_zone_reading`|`false`|Specifies whether to prioritize sending queries to the storage nodes in the same zone.<br/>When set to `true`, the query is sent to the storage nodes in the same zone. If reading fails in that Zone, it will decide based on `stick_to_intra_zone_on_failure` whether to read the leader partition replica data from other Zones. |
+        |`spec.graphd.config.stick_to_intra_zone_on_failure`|`false`|Specifies whether to stick to intra-zone routing if unable to read the requested partitions in the same zone. When set to `true`, if reading fails in that Zone, it does not read data from other Zones.|
+
+        ???+ note "Learn more about Zones in NebulaGraph Operator"
+
+            **Understanding NebulaGraph's Zone Feature**
+
+            NebulaGraph utilizes a feature called Zones to efficiently manage its distributed architecture. Each Zone represents a logical grouping of Storage pods and Graph pods, responsible for storing the complete graph space data. The data within NebulaGraph's spaces is partitioned, and replicas of these partitions are evenly distributed across all available Zones. The utilization of Zones can significantly reduce inter-Zone network traffic costs and boost data transfer speeds.
```

Comment:
1. `The utilization of Zones can significantly reduce inter-Zone network traffic costs and boost data transfer speeds. ` This is the benefit of intra-zone-reading. We miss one more benefit here. Since we the replicas of a partition now reside in different zones, we improve the availability.